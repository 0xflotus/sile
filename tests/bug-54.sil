\begin[papersize=a4, class=book]{document}
\set[parameter=document.parindent, value=10mm]
\set[parameter=current.parindent, value=10mm]
First line \par
First word, first paragraph.

First word, second paragraph.
\end{document}
