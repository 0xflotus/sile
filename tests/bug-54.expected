<tests/bug-54.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	49.32284
My 	37.33214
Set font 	Gentium;10;200;regular;normal;;LTR
T	38 73 82 83 84	(First)
Mx 	70.75468
T	76 73 78 69	(line)
Mx 	49.32284
My 	49.33214
T	38 73 82 83 84	(First)
Mx 	70.74638
T	87 79 82 68	(word)
Mx 	91.99638
T	12	(,)
Mx 	96.95508
T	70 73 82 83 84	(first)
Mx 	116.70870
T	80 65 82 65 71 82 65 80 72	(paragraph)
Mx 	159.38936
T	14	(.)
Mx 	49.32284
My 	61.33214
T	38 73 82 83 84	(First)
Mx 	70.74511
T	87 79 82 68	(word)
Mx 	91.99511
T	12	(,)
Mx 	96.95254
T	83 69 67 79 78 68	(second)
Mx 	128.21857
T	80 65 82 65 71 82 65 80 72	(paragraph)
Mx 	170.89923
T	14	(.)
[1] Mx 	207.41759
My 	553.73265
T	17	(1)
End page
Finish

